include/Utilities.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <memory>
+#include <string>
+#include <map>
+
+using std::cout;
+using std::cin;
+using std::endl;
+using std::string;
+using std::unique_ptr;
+using std::vector;
+using std::make_unique;
+using std::move;
+
+class Piece;
+
+using board_vec = vector<vector<unique_ptr<Piece>>>;
+using board_vec_const = const vector<vector<unique_ptr<Piece>>>;
+
+using board_vec_ref = vector<vector<unique_ptr<Piece>>>&;
```

Comment:
Also, it's good practice to mark types with a `_t` suffix (like `size_t`). That way the reader immediately knows it's a type